sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/CommunicationIdentifier.java

```diff
@@ -24,7 +24,7 @@ public static CommunicationIdentifier fromRawId(String rawId) {
         }
 
         if (rawId.startsWith("4:")) {
-            return new PhoneNumberIdentifier(rawId.substring("4:".length()));
+            return new PhoneNumberIdentifier("+" + rawId.substring("4:".length()));
```

Comment:
revert all changes in the CommunicationIdentifier, PhoneNumberIdentifier, CommunicationIdentifierTests java files back, this fix should be included into the patch release